aapcs64/aapcs64.rst

```diff
@@ -521,6 +538,35 @@ Elements in a short vector are numbered such that the lowest numbered element (e
 
 A language binding may define extended types that map directly onto short vectors. Short vectors are not otherwise created spontaneously (for example because a user has declared an aggregate consisting of eight consecutive byte-sized objects).
 
+Scalable Vectors
+----------------
+
+Like a short vector (see `Short Vectors`_), a scalable vector is a
+machine type that is composed of repeated instances of one fundamental
+integral or floating-point type. The number of bytes in the vector is
+always VG×8, where VG is a runtime value determined by the execution
+environment. VG is an even integer greater than or equal to 2; the ABI
+does not define an upper bound. VG is the same for all scalable vector
```

Comment:
```suggestion
does not define an upper bound. VG is the same for all scalable vectors
```

Reply:
Fixed by expanding "scalable vector and predicate types" to "scalable vector types and scalable predicate types".